pkg/downloader/chart_downloader.go

```diff
@@ -181,8 +181,10 @@ func (c *ChartDownloader) ResolveChartVersion(ref, version string) (*url.URL, er
 		c.Options = append(
 			c.Options,
 			getter.WithURL(rc.URL),
-			getter.WithTLSClientConfig(rc.CertFile, rc.KeyFile, rc.CAFile),
 		)
+		if rc.CertFile != "" || rc.KeyFile != "" || rc.CAFile != "" {
+			getter.WithTLSClientConfig(rc.CertFile, rc.KeyFile, rc.CAFile)
```

Comment:
should this be appended to `c.Options` like it was before?

Reply:
fixed in bf5c0ae